packages/grafana-ui/.storybook/.babelrc.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
only thing that makes me a little queasy is this is a large amount of custom babel settings, buuuuuut i'm assuming there's no way round it? 😅

Reply:
Yeah, I'm not too keen on it either. I was hoping the storybook default babelrc would handle things but without this storybook cannot compile `grafana/ui`. Throws lots of errors related to not understanding typescript?!

```SyntaxError: /Users/jackwestbrook/Projects/grafana/packages/grafana-ui/src/slate-plugins/slate-prism/options.tsx: TypeScript 'declare' fields must first be transformed by @babel/plugin-transform-typescript.
If you have already enabled that plugin (or '@babel/preset-typescript'), make sure that it runs before any plugin related to additional class features:
 - @babel/plugin-proposal-class-properties
 - @babel/plugin-proposal-private-methods
 - @babel/plugin-proposal-decorators
  57 |   implements OptionsFormat
  58 | {
> 59 |   declare readonly onlyIn: (node: Node) => boolean;
     |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  60 |   declare readonly getSyntax: (node: Node) => string;
  61 |   declare readonly renderDecoration: (
  62 |     {```

Let me have a dabble and see if I can figure out a better approach. 😄

Reply:
@ashharrison90 please see https://github.com/grafana/grafana/pull/49793/commits/f55d927f4d4150be1864a373ac7068c4592af2f5

Reply:
@jackw anything that says `turbo-build` sounds good to me 😂

Reply:
Shame it broke the build of core. I'll fix it up shortly.